index.html

```diff
@@ -195,6 +195,30 @@ <h2>About
         </ul>
       </div>
     </div>
+    <div class="contact-me">
+      <h2 class="heading">Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Great job. please your form still doesnt submit succesfully. like the previous reviewer noted, the Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). kindly create a formspree account and chose new form. then update your forms action(the `form_id`) with the form endpoint in your Formspree account. you can see how to do that [here](https://formspree.io/html/)

Reply:
@Edgahkipkemoi please write me on slack @tufoin